src/apps/frontend/pages/authentication/login/login-form.tsx

```diff
@@ -24,8 +24,8 @@ const LoginForm: React.FC<LoginFormProps> = ({ onError, onSuccess }) => {
 
   return (
     <>
-      <h2 className="mb-9 text-2xl font-bold text-black dark:text-white sm:text-title-xl2">
-        Sign In
+      <h2 className="mb-9 text-2xl font-bold text-black sm:text-title-xl2">
```

Comment:
- Create `H2` component with `text-2xl font-bold text-black sm:text-title-xl2`, and ensure we use `H2` component other places without CSS classes.
- I think we need to add margin here between header, and body. Create a card component with such that below markup is:


```
<Card>
  <CardHeader>Log In</CardHeader>
  <CardBody>form ...</CardBody>
</Card>
```

```
CardHeader = ({title: ReactNode}) => {
  <H2>{title}</H2>
}
```

`Card` should use vertical gap to add appropriate gap between header and body. It should also have appropriate padding in it.

Reply:
Let's try that.